pkg/ast/ast.go

```diff
@@ -124,6 +128,20 @@ type ContextCondition struct {
 	Subject Subject
 	Where   *WhereClause
 }
+
+func (a *ContextCondition) String() string {
+	var out bytes.Buffer
+	out.WriteString("{")
+	if !types.IsNilInterface(a.Subject) {
+		out.WriteString(fmt.Sprintf("Subject: %s, ", a.Subject.String()))
```

Comment:
should be same as seal syntax, i.e. something like:
```suggestion
		out.WriteString(fmt.Sprintf("subject %s ", a.Subject.String()))
```

should be `subject user foo` or `subject group bar`

Reply:
done